pepper-framework/src/main/java/org/corpus_tools/pepper/modules/PepperModuleProperty.java

```diff
@@ -21,71 +21,53 @@
 import java.io.Serializable;
 
 /**
- * The class {@link PepperModuleProperty} offers a possibility to describe a
- * property, which is used by a specific {@link PepperModule}. A list of
- * {@link PepperModuleProperty} objects is requested by Pepper to give an
- * overview of properties to the user. T specifies the type of this property.
- * This class contains:
- * <ol>
- * <li>the name of this property</li>
- * <li>a description of its sense to present it to the user</li>
- * <li>a field determining if this property is required by the corresponding
- * {@link PepperModule} object</li>
- * </ol>
+ * With properties, the conversion process done by a Pepper module can be
+ * customized. A property in general is an attribute value pair, with a unique
+ * name and a value influencing the module's behavior. It is possible to add a
+ * default value in case the value was not initialized by the Pepper workflow
+ * file. A further flag can be used to set the property as required, which
+ * means, the property must be set to run the container module. Each property
+ * should contain a description, to help the user understanding how to use this
+ * property.
  * 
- * @author Florian Zipser
+ * A property is created in a fluent way. See the following sample:
  * 
+ * <pre>
+ * PepperModuleProperty.create().withName("MyProp").withClass(Boolean.class).withDefaultValue(true)

```

Comment:
I'd either give the sample as `PepperModuleProperty.create().withName("MyProp").withClass(Boolean.class).withDefaultValue(defaultValue)...` so that it's clear that a defined default value can be passed, and not just record the fact that a property can have a default value.

Or just do a typed pseudo code sample, such as 
`PepperModuleProperty.create().withName(String name).withClass(Clazz.class).withDefaultValue(Object defaultValue).withDescription(String description).isRequired(boolean bool).build()`.